lib/mountable_file_server/metadata.rb

```diff
@@ -0,0 +1,45 @@
+require 'dimensions'
+
+module MountableFileServer
+  class Metadata
+    attr_reader :size, :content_type, :height, :width
+
+    def initialize(size:, content_type:, width: nil, height: nil)
```

Comment:
According to Sandi Metz (POODR) it's preferred to use an argument hash over multiple ordered arguments.

- The method is order independent.
- You don't have to remember what argument stands for what when you call the method.
- The method signature stays the same for callers when you make changes.

I tend to agree with her and would like to try it. I picked Ruby keyword arguments since they represent all the strengths without having to deal with a hash myself.